scripts/local_testnet/README.md

```diff
@@ -1,11 +1,11 @@
 # Simple Local Testnet
 
-These scripts allow for running a small local testnet with multiple beacon nodes and validator clients.
+These scripts allow for running a small local testnet with multiple beacon nodes and validator clients and a geth execution client.
 This setup can be useful for testing and development.
 
 ## Requirements
 
-The scripts require `lcli` and `lighthouse` to be installed on `PATH`. From the
+The scripts require `lcli`, `lighthouse`, `geth`, `bootnode` to be installed on `PATH`. From the
 root of this repository, run:
 
 ```bash
```

Comment:
For MacOS, we also need GNU `sed` and GNU `grep` for the scripts, so may be worth adding to docs?

````
### MacOS Users

Install GNU `sed` and GNU `grep`, and add them both to `PATH`, e.g.

```
brew install gnu-sed grep
echo "export PATH=\$PATH:$(brew --prefix)/opt/grep/libexec/gnubin" >> ~/.bash_profile
echo "export PATH=\$PATH:$(brew --prefix)/opt/gnu-sed/libexec/gnubin" >> ~/.bash_profile
```
````

Reply:
Will add a note. Thanks for raising